.travis.yml

```diff
@@ -6,23 +6,58 @@ before_install:
   - unset _JAVA_OPTIONS
 env:
   global:
-    - GRADLE_OPTS="-Xmx256m"
+    - GRADLE_OPTS='-Xmx256m'
+    - BUILD_CACHE_DIR="$HOME/build-cache"
+    - TESTS_FILE_NAME="$TRAVIS_BUILD_DIR/tests.txt"
+    - TESTS_FILES_PREFIX='tests'
+    - OP_VERSION_PROPS_DISABLED=false
```

Comment:
`OP_VERSION_PROPS_DISABLED=true`

Reply:
Was wondering how the local testing passed when false is set. Perhaps [this check](https://github.com/salesforce/TransmogrifAI/blob/master/gradle/version-properties.gradle#L19) is validating if the property is defined (or) not.